db/fixtures/miq_event_definitions.csv

```diff
@@ -147,6 +147,7 @@ vm_remote_console_connected,VM Remote Console Connected,Default,vm_operational
 #
 request_storage_scan,Datastore Analysis Request,Default,storage_operational
 storage_scan_complete,Datastore Analysis Complete,Default,storage_operational
+db_failover_executed,Datastore Failover Executed,Default,storage_operational
```

Comment:
we discussed icon=>events assignment with @epwinchell Eric suggested to use the same existing ```fa fa-database```  for ```Database``` events and he will change icon for ```Datastore``` events later